test/llvm/PREDIFF

```diff
@@ -8,6 +8,12 @@ PLATFORM=`$CHPL_HOME/util/chplenv/chpl_platform.py --host`
 COMPILER=`$CHPL_HOME/util/chplenv/chpl_compiler.py --host`
 FILECHECK=${CHPL_HOME}/third-party/llvm/install/${PLATFORM}-${COMPILER}/bin/FileCheck
 
+if [ ! -x $FILECHECK ]; then
+    PREFERRED_LLVM_VERS=`cat ${CHPL_HOME}/third-party/llvm/LLVM_VERSION`
+    LLVM_CONFIG=`${CHPL_HOME}/third-party/llvm/find-llvm-config.sh $PREFERRED_LLVM_VERS`
+    FILECHECK=${LLVM_CONFIG//llvm-config/FileCheck}
```

Comment:
I assume this is intended to replace llvm-config with FileCheck?
I can't find this syntax in the Make manual. I'd assume it'd be documented here 
  https://www.gnu.org/software/make/manual/html_node/Text-Functions.html
Anyway would it be clearer to use ${subst llvm-config,FileCheck,${LLVM_CONFIG}} (but I havn't tested that)?

Reply:
It's funny that we both made the same mistaken assumption.  I first wrote this in Makefile syntax, but that didn't work because it is a bash script instead.

Reply:
haha OK right you are now it's in the bash manual. Thanks. Not sure why I thought we were writing Makefiles (I guess it just looks Make-y).